README.md

```diff
@@ -190,15 +177,31 @@ pub fn shave_all(yaks: usize) -> usize {
 tracing = "0.1"
 ```
 
-Note: Libraries should *NOT* call `set_global_default()`, as this will cause
-conflicts when executables try to set the default later.
+Note: Libraries should *NOT* install a subscriber by using a method than calls
+`set_global_default()`, as this will cause conflicts when executables try to
```

Comment:
I think we should at least discuss the _existence_ of  `set_global_default` and what it does before this warning. If the first example is changed to show `tracing_subscriber::init`, then users will read this note before even knowing that there is a function called  `set_global_default`.

Perhaps another example after the first one, explaining that `tracing_subscriber::init()` uses `set_global_default`, the generalized way of setting a global default subscriber?

Reply:
Added some better joining language between the examples to address this. I think one simple example and one complex example is enough?